var/spack/repos/builtin/packages/py-parso/package.py

```diff
@@ -22,4 +23,5 @@ class PyParso(PythonPackage):
     depends_on('python@3.6:', type=('build', 'run'), when='@0.8.1:')
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'), when='@0.6.1:')
     depends_on('python@2.6:2.8,3.3:', type=('build', 'run'), when='@0.4.0:')
+    depends_on('python@3.6:', type=('build', 'run'), when='@0.8.2:')
```

Comment:
This is already on the line above, you can remove this

Reply:
Thanks!